.github/workflows/build.yml

```diff
@@ -9,10 +9,20 @@ jobs:
       fail-fast: false
       matrix:
         config:
-        - {os: "ubuntu-latest", url: "https://github.com/casey/just/releases/download/v0.5.8/just-v0.5.8-x86_64-unknown-linux-musl.tar.gz", name: "just", pathInArchive: "just" }
-        - {os: "macos-latest", url: "https://github.com/casey/just/releases/download/v0.5.8/just-v0.5.8-x86_64-apple-darwin.tar.gz", name: "just", pathInArchive: "just" }
+          - {
```

Comment:
This was just formatting that ran automatically, but I think it looks nicer anyway, so I kept it